sdk/eventhub/Azure.Messaging.EventHubs/samples/Program.cs

```diff
@@ -15,6 +15,16 @@ namespace Azure.Messaging.EventHubs.Samples
     ///
     public static class Program
     {
+        private static string[] extraOptionsForSamples = new[]
```

Comment:
nit:  If you would be so kind, for consistency, please add single line summary doc comments for these.